src/components/Toolbar/Index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+    StyleSheet,
+    ToolbarAndroid,
+    ToolbarAndroidAction
+} from 'react-native';
+import * as React from 'react';
+import icons, {
+    images
+} from '../../assets/index';
+
+/*Only one button is permitted on the left
+it must either be a menu button which has a 'burger' icon
+or a back button which has an arrow icon*/
+export enum LeftButtonTypes {
+    menu = "menu",
+    back = "back"
+}
+/*Multiple buttons can be added to the right side of the toolbar
+if an icon isn't specified the title will be displayed in its place*/
+interface RightButtonType {
+    title: string;
+    icon ? : any;
+    action: () => void;
+}
+/*This specifies the toolbar properties we intend to allow the pages to control
+currently the title, left button type (menu or back) and right buttons can be specified*/
+interface ToolbarProps {
+    title: string;
+    leftButtonType: LeftButtonTypes;
+    rightButtons: Array < RightButtonType > ;
+}
+
+export default class Toolbar extends React.Component < ToolbarProps, {} > {
+    constructor(props: ToolbarProps) {
+    super(props);
```

Comment:
I think once we have a linter we'll have reformat some of the code! (This should be indented)

Reply:
I put it through that online formatter. I guess I shouldn't blindly trust it

Reply:
Enoch is almost done with the linting for this project, and the linter does have a `--fix` command to fix what it can. Some changes you will have to make yourself unfortunately